addons/account_edi_facturx/data/facturx_templates.xml

```diff
@@ -209,23 +208,23 @@
                         <!-- Summary. -->
                         <ram:SpecifiedTradeSettlementHeaderMonetarySummation>
                             <ram:LineTotalAmount
-                                t-att-currencyID="currency.name"
-                                t-esc="format_monetary(record.amount_untaxed, currency)"/>
+                                t-att-currencyID="record.currency_id.name"
```

Comment:
What is the idea behind currency -> record.currency_id, to make clear it is not company_currency?

Reply:
The `currency` variable wasn't very necessary imo